xbmc/pvr/recordings/PVRRecordingsPath.cpp

```diff
@@ -230,3 +239,17 @@ size_t CPVRRecordingsPath::GetDirectoryPathPosition() const
   }
   // unreachable
 }
+
+void CPVRRecordingsPath::EscapeSegment(std::string &strSegment)
```

Comment:
well, i was wrong, i was misinterpreting what it does when i was first reading the code. will change, thanks for the hint.